tests/runtests.pl

```diff
@@ -15,17 +15,20 @@
 my $output = `id`;
 $output =~ /uid=\d+\((\w+)\).*/;
 my $unix_user = $1;
-$output =~ /context=(\w+):(\w+):(\w+)/;
+$output =~ /context=(\w+):(\w+):(\w+):(\w+)-([\w:\.]+)/;
 my $selinux_user = $1;
 my $selinux_role = $2;
 my $selinux_type = $3;
+my $selinux_sen = $4;
```

Comment:
The SELinux MLS/MCS field is a range, not a single sensitivity level and category bitmap.  While the significance of the range is configurable via policy constraints, ever SELinux policy I have ever seen treats the low end of the range as the effective (hence the "eff") label and the upper end of the range as the cleared label.

Take the following SELinux label as an example:

    user_u:role_r:type_t:s0:c1-s2:c3,c4

In this particular case the effective sensitivity level and category bitmap is:

    s0:c1

... while the process (side note: ranged objects rarely, if ever, make sense) is cleared to the following sensitivity level and category:

    s2:c3,c4

As an additional note, sometimes you may not see an explicitly ranged MLS/MCS field, as in the example below; this doesn't mean the range doesn't exist, it simply means that the effective and cleared labels are the same.

    user_u:role_r:type_t:s5:c6,c7

We can talk about this further if you like, but I think that is beyond the scope of this code review.